packages/components/src/components/Tooltip/TooltipContent.stories.tsx

```diff
@@ -0,0 +1,134 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { type Meta, type StoryObj } from '@storybook/react';
+
+import { Button } from '../Button/Button';
+
+import { Tooltip, TooltipContent } from './Tooltip';
+
+type Story = StoryObj<typeof TooltipContent>;
+const meta: Meta<typeof TooltipContent> = {
+    component: TooltipContent,
+    title: 'Components/Tooltip/Content',
```

Comment:
```suggestion
    title: 'Components/Tooltip',
```
Then it merges the 2 stories file

Reply:
this breaks/removes the controls for TooltipContent that was the reason i out a second file